utilities/include/vec3.hpp

```diff
@@ -1,212 +1,59 @@
 #ifndef VEC3_HPP
 #define VEC3_HPP
 
-
 #include <iostream>
```

Comment:
Change the `iostream` include to `ostream`.

Reply:
That's not needed anyway. I'll remove the header